src/main/java/com/halo/core_bridge/api/jobposting/controller/JobPostingController.java

```diff
@@ -0,0 +1,37 @@
+package com.halo.core_bridge.api.jobposting.controller;
+
+
+import com.halo.core_bridge.api.jobposting.model.dto.JobPostingDto;
+import com.halo.core_bridge.api.jobposting.service.JobPostingService;
+import com.halo.core_bridge.common.model.BaseResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/job-postings")
+public class JobPostingController {
+    private final JobPostingService jobPostingService;
+
+    //채용공고 등록
+    @PostMapping
+    public BaseResponse<JobPostingDto.DetailResponse> createJobPosting(
+            @RequestBody JobPostingDto.CreateRequest request
+    ) {
+       return BaseResponse.success(jobPostingService.save(request));
+    }
+
+    //채용공고 전체 목록 조회
+    @GetMapping
+    public BaseResponse<List<JobPostingDto.ListResponse>> getAllJobPostings() {
+        return BaseResponse.success(jobPostingService.getList());
+    }
```

Comment:
Reponse 응답을 내려줄 때 ResponseEntity로 한번 더 감싸 주는 방법이 더 좋을 것 같아요!
지금 구조에서는 모든 응답이 기본적으로 `200 OK`로 내려가게 되는데,
이렇게 되면 REST API 설계 원칙상 성공/실패 구분이 명확하지 않아 **좋은 API 설계**라고 보기 어려워요.

`ResponseEntity`를 사용하면 `ok()`, `created()`, `badRequest()` 등 다양한 상태 코드를 지정할 수 있어서 더 RESTful한 방식으로 응답을 내려줄 수 있어요!

`ResponseEnity`나 REST API 응답 구조 등 관련해서 한번 찾아보시면 도움이 많이 될 것 같아요!

```suggestion
    @GetMapping
    public ResponseEntity<BaseResponse<List<JobPostingDto.ListResponse>>> getAllJobPostings() {
    
        return ResponseEntity.ok(BaseResponse.success(jobPostingService.getList()));
    }
```